lib/api/core/hotelClerk.js

```diff
@@ -790,3 +793,61 @@ function subscribeToRoom (roomId, requestObject, context) {
 
   return deferred.promise;
 }
+
+
+/**
+ * Authentication token white-list.
+ *
+ * Maintain a list of valid token.
+ * When a token is invalidated, it remove all room subscriptions from connexion
```

Comment:
Should be

    When a token is invalidated, it removes all room subscriptions from connection.